src/theme/NavbarItem/DocsVersionDropdownNavbarItem.js

```diff
@@ -4,7 +4,7 @@ import {
   useActiveDocContext,
 } from "@docusaurus/plugin-content-docs/client";
 import { useDocsPreferredVersion } from "@docusaurus/theme-common";
-import { useDocsVersionCandidates } from "@docusaurus/theme-common/internal";
+import { useDocsVersionCandidates } from '@docusaurus/plugin-content-docs/client';
```

Comment:
I could not find a reference to this in the release notes, I think it falls through the semver cracks as an "internal" api.